src/zap.c

```diff
@@ -4626,7 +4626,7 @@ short exploding_wand_typ;
         case ZT_COLD:
             new_doormask = D_NODOOR;
             see_txt = "The door freezes and shatters!";
-            sense_txt = "feel cold.";
+            hear_txt = "a deep cracking sound.";
```

Comment:
can we broke door after that (cold damage) because use cold? If answer is "Yes" then how we understand that cracking sound is craking of cold from this text?